src/main/java/org/htwkvisu/controller/ApplicationController.java

```diff
@@ -22,6 +24,20 @@
 public class ApplicationController implements Initializable {
 
     @FXML
+    private NumericTextField pixelDensityTextField;
+
+    private static final int DEFAULT_PIXEL_DENSITY = 30;
+    private static final int MAX_NUMERIC_FIELD_LENGTH = 6;
+
+    @FXML
+    private NumericTextField minScoringTextField;
+    private static final int DEFAULT_MIN_SCORING_VALUE = 0;
```

Comment:
Ich habe es nicht hinbekommen in der FXML die NumericFields in extra FXML-Files auszulagern, damit ich einen andere Controller verwenden kann.
Deswegen muss es derzeit in dem ApplicationController bleiben.